api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -9,26 +9,6 @@
  */
 package org.openmrs.api;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNotSame;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.openmrs.test.TestUtil.containsId;
-
-import java.sql.SQLException;
-import java.util.ArrayList;
```

Comment:
It would be better if these imports were not rearranged in this pull request.

Reply:
Let me fix this , but it's been done by the IDE code formatter